### PR TITLE
Tp/b2btrialmessages

### PR DESCRIPTION
**Pull request by tom-parker**

### CODE REVIEW

File: src/client/bottom-slot.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ const { generateMessageEvent, listen } = require('./utils');
 
 const BANNER_CLASS = 'n-messaging-banner';
 const BANNER_ACTION_SELECTOR = '[data-n-messaging-banner-action]';
-const BANNER_BUTTON_SELECTOR = `.${BANNER_CLASS}__button`;
+const BANNER_IMPERATIVE_ACTION_SELECTOR = `.${BANNER_CLASS}__action a`;
```
Comment (liamkeaton) [inline src/client/bottom-slot.js]: Will this cause issues with tracking when it's not an `anchor` but a `button`, for instance https://github.com/Financial-Times/n-messaging-client/blob/master/templates/partials/bottom/onboarding-app.html#L19

Comment (tom-parker) [reply src/client/bottom-slot.js]: I think that one might have been caught with the `BANNER_ACTION_SELECTOR` but this does seem a bit weird. Maybe I'm better off merging the button and link selections

**Comment by kavanagh** [comment]
This PR was merged (indirectly via our Renovatebot PR) onto next-article without even a description or understandable commit message. Review and descriptions and reviews on this repo need to be better than this if we're going to work in this way.

**Comment by tom-parker** [comment]
Apologies, descriptions and commit messages aside, do you think there's a better process for this?

**Comment by kavanagh** [comment]
I think info on PRs will help a lot. I don't know a better process yet because I don't really know what we have (I'm slightly out of the loop). Let's discuss in email/person.

**Pull request merged**